README.md

```diff
@@ -523,8 +524,8 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :bulb: [50 Shades of Go: Traps, Gotchas, and Common Mistakes for New Golang Devs](http://devs.cloudimmunity.com/gotchas-and-common-mistakes-in-go-golang/)
 - :video_camera: [Golang for begineers - YouTube playlist](https://www.youtube.com/playlist?list=PLRAV69dS1uWSR89FRQGZ6q9BR2b44Tr9N)
 - :video_camera: [Let's go with golang - YouTube playlist](https://www.youtube.com/playlist?list=PLRAV69dS1uWQGDQoBYMZWKjzuhCaOnBpa)
+- :video_camera: [let's go with golang series - YouTube playlist](https://www.youtube.com/watch?v=C81eCx1KSYg&list=PL-Jc9J83PIiGhwxgJGiYxGG7zChL1PmAx)
```

Comment:
The playlist link should be https://www.youtube.com/playlist?list=PL-Jc9J83PIiGhwxgJGiYxGG7zChL1PmAx